standard_template/standard/annex-b.adoc

```diff
@@ -0,0 +1,203 @@
+[appendix]
+:appendix-caption: Annex
+== Coverage Examples ( {Informative} )
+
+### Vertical Profile Coverage
+
+```json
```

Comment:
Global comment: we should refactor the JSON examples in standalone files which we can ref accordingly.  This will reduce clutter in the AsciiDoc's, as well as allow for linting/validation of CoverageJSON examples for quality/consistency.

Reply:
@tomkralidis PR done, Jon, Ethan and Steve happy with re-factoring examples to simplify AsciiDoc.